test/test_browser.py

```diff
@@ -5521,6 +5521,22 @@ def test_full_js_library_strict(self):
   def test_audio_worklet(self, args):
     self.btest_exit('webaudio/audioworklet.c', emcc_args=['-sAUDIO_WORKLET', '-sWASM_WORKERS', '-DTEST_AND_EXIT'] + args)
 
+  # Test that when using audio worklets, it is possible to use the locateFile() machinery to reroute where the .aw.js file is loaded from.
+  @requires_sound_hardware
+  def test_audio_worklet_locatefile(self):
+    # Create a locateFile() function that loads .aw.js from a subdirectory.
+    create_file('shell.html', read_file(path_from_root('src/shell.html')).replace('var Module = {', '''
+    var Module = {
+      locateFile: function(path, prefix) {
+        return path.includes('.aw.js') ? 'subdir/' + path : path;
+      },
+    '''))
```

Comment:
I don't think this requires a custom shell.html.

I think you can just put the `Module = { ... }` stuff in a `--pre-js` file there.